core/che-core-db/src/main/java/org/eclipse/che/core/db/TracingDataSource.java

```diff
@@ -85,7 +85,7 @@ public Logger getParentLogger() throws SQLFeatureNotSupportedException {
   }
 
   public static DataSource wrapWithTracingIfEnabled(DataSource dataSource) {
-    return Boolean.valueOf(System.getenv("CHE_TRACING_ENABLED"))
+    return Boolean.valueOf(System.getenv("CHE_DB_TRACING_ENABLED"))
```

Comment:
I think we should tune this specific trace in this way (with the environment variable).
Agree that db tracing is quite talkative and it makes sense to remove/disable it or make it configurable. 
We can do that with jaeger configuration.
1. To disable we can use Jaeger configuration https://www.jaegertracing.io/docs/1.10/sampling/#collector-sampling-configuration
2. With named parameter
3. Remove it at all.

I would say 3->2->1 would be my vote list. @metlos @ibuziuk @garagatyi  wdyt?
Anyway I think it's worth of separate issue/PR

Reply:
I agree with those options and the order; I mostly didn't want to remove it wholesale since I didn't fully understand its purpose.

Reply:
I generally don't think that tracing each database access is worthwhile, since we're not getting very actionable information (e.g. an `UPDATE` takes a long time -- what do we do about it). It would be more useful to track things like number database operations, average time, etc.

Reply:
@amisevsk  can you comment your opinion at  https://github.com/eclipse/che/issues/12851?

Reply:
Not sure it is purpose of this PR to remove this completely. Every distinct operation trace doesn't say much, but all of them overall do say that we have HUGE amount of DB calls on every workspace operation.

Reply:
ok let's add an ability to disable db traces in this pr.
maybe the condition should be like: 
```Boolean.valueOf(System.getenv("CHE_DB_TRACING_ENABLED")) && Boolean.valueOf(System.getenv("CHE_TRACING_ENABLED"))``` wdyt?